pkg/schema/annotations.go

```diff
@@ -402,6 +512,66 @@ func getTypeFromAnnotations(anns []Annotation, pos *filepos.Position) (Type, err
 	return typeFromAnn, nil
 }
 
+type documentation struct {
+	description        string
+	exampleDescription string
+	exampleYAML        interface{}
+}
+
+func setDocumentationFromAnns(docAnns []Annotation, typeOfValue Type) error {
+	var documentationInfo documentation
+	for _, a := range docAnns {
+		switch ann := a.(type) {
+		case *DescriptionAnnotation:
+			documentationInfo.description = documentationInfo.description + ann.description
```

Comment:
I am curious about this line. It looks like we are concatenating all the `description` annotations values together here into one string. But I was under the impression that only the first annotation will be used if there are multiple of the same type. Whats the story here?

Reply:
That's a good point, I want to maintain the same behavior as before, so if we changed it to `documentationInfo.description = ann.description` would do the trick?